README.md

```diff
@@ -132,6 +133,33 @@ for logger_name in ('snowflake.snowpark', 'snowflake.connector'):
     logger.addHandler(ch)
 ```
 
+## Reading and writing to pandas DataFrame
+
+Snowpark Python API supports reading from and writing to a pandas DataFrame via the [to_pandas][to_pandas] and [write_pandas][write_pandas] commands. 
+
+To use these operations, ensure that pandas is installed in the same environment. You can install pandas alongside Snowpark Python by executing the following command:
+```bash
+pip install "snowflake-snowpark-python[pandas]"
+```
+Once pandas is installed, you can convert between a Snowpark DataFrame and pandas DataFrame as follows: 
+```python
+df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
+# Convert Snowpark DataFrame to pandas DataFrame
+pandas_df = df.to_pandas() 
+# Write pandas DataFrame to a Snowflake table and return Snowpark DataFrame
+snowpark_df = session.write_pandas(pandas_df, "new_table", auto_create_table=True)
+```
+
+Snowpark pandas API also supports writing to pandas: 
+```python
+import modin.pandas as pd
+df = pd.DataFrame([[1, 2], [3, 4]], columns=["a", "b"])
+# Convert Snowpark pandas DataFrame to pandas DataFrame
+pandas_df = df.to_pandas() 
+```
+
+Note that the above Snowpark pandas commands will work if Snowpark is installed with the `[modin]` option, the additional `[pandas]` installation is not required.
```

Comment:
```suggestion
Note that the above Snowpark pandas commands will work if Snowpark is installed with the `[modin]` option, and the additional `[pandas]` installation is not required.
```

Reply:
ah rip, it auto merged before I saw this. I'll add it to a future PR.